grails-app/domain/com/careerfly/user/User.groovy

```diff
@@ -0,0 +1,21 @@
+package com.careerfly.user
+
+class User {
+    String firstName
+    String lastName
+    String email
+    String password
+    Gender gender
+    Date dateCreated
+    Date lastUpdated
+
+    static constraints = {
+        firstName(blank: false, matches: "[a-zA-Z]+")
+        lastName(blank: false, matches: "[a-zA-Z]+")
+        email(email: true,unique: true)
+        password(blank: false , matches: "[a-zA-Z0-9]+")
+    }
+}
+enum Gender {
+    MALE, FEMALE
```

Comment:
@akshitagangrade @PraveenTahalyani @apoorvsrivastava1116 We have to use `id` based implementation like we seen in the class.